Office_Space_Allocation/person.py

```diff
@@ -0,0 +1,16 @@
+class Person(object):
+	def __init__(self,person_name,person_type):
+		self.person_name = person_name
+		self.person_type = person_type
+
+class Fellow(Person):
+	def __init__(self,person_name,person_type = "fellow"):
+		super(Fellow,self).__init__(person_name,person_type)
+
+class Staff(Person):
+	def __init__(self,staff_name,staff_type = "staff"):
+		super(Staff, self).__init__(staff_name,staff_type)
+				
+new_fellow = Fellow("Sharon")
```

Comment:
Why do you have this in your code? The user is the one who is supposed to perform these actions..